spec/models/organisation_spec.rb

```diff
@@ -27,6 +27,22 @@
         .to raise_error(ActiveRecord::RecordInvalid, "Validation failed: Provider type #{I18n.t('validations.organisation.provider_type_missing')}")
     end
 
+    context "with parent/child association" do
+      let(:child_organisation) { FactoryBot.create(:organisation, name: "DLUHC_Child") }
```

Comment:
```suggestion
      let(:child_organisation) { FactoryBot.create(:organisation, name: "DLUHC Child") }
```

nit: underscored name